sqldb/sqlc/migrations/000007_graph.up.sql

```diff
@@ -208,3 +208,77 @@ CREATE TABLE IF NOT EXISTS channel_extra_types (
 CREATE UNIQUE INDEX IF NOT EXISTS channel_extra_types_unique ON channel_extra_types (
     type, channel_id
 );
+
+/* ─────────────────────────────────────────────
+   channel policy data tables
+   ─────────────────────────────────────────────
+*/
+
+CREATE TABLE IF NOT EXISTS channel_policies (
+    -- The db ID of the channel policy.
+    id INTEGER PRIMARY KEY,
+
+    -- The protocol version that this update was gossiped on.
+    version SMALLINT NOT NULL,
+
+    -- The DB ID of the channel that this policy is referencing.
+    channel_id BIGINT NOT NULL REFERENCES channels(id) ON DELETE CASCADE,
+
+    -- The DB ID of the node that created the policy update.
+    node_id BIGINT NOT NULL REFERENCES nodes(id) ON DELETE CASCADE,
```

Comment:
I was wondering if instead of referencing the ids so we can join, we'd reference the scid and the node1 and node2 where we'd always make sure node1 is the originator of the channel? References are cheap in SQL and they also do not duplicate data and that way we could avoid needing to fetch the channel (joined with nodes) on a policy update. wdyt?